src/components/NurseryWithdrawals/WithdrawalDetails/sections/Photos.tsx

```diff
@@ -1,15 +1,79 @@
-import { Box, Typography } from '@mui/material';
+import { Box, Typography, useTheme } from '@mui/material';
 import strings from 'src/strings';
+import { useEffect, useState } from 'react';
+import { getWithdrawalPhotosList } from 'src/api/tracking/withdrawals';
+import useSnackbar from 'src/utils/useSnackbar';
+import ViewPhotosModal from 'src/components/common/ViewPhotosModal';
 
-type PhotosSectionProps = {};
+const NURSERY_WITHDRAWAL_PHOTO_ENDPOINT = '/api/v1/nursery/withdrawals/{withdrawalId}/photos/{photoId}';
+
+type PhotosSectionProps = {
+  withdrawalId?: number;
+};
+
+export default function Photos({ withdrawalId }: PhotosSectionProps): JSX.Element {
+  const theme = useTheme();
+  const [snackbar] = useState(useSnackbar());
+  const [photoUrls, setPhotoUrls] = useState<string[]>([]);
+  const [photosModalOpened, setPhotosModalOpened] = useState(false);
+  const [selectedSlide, setSelectedSlide] = useState(0);
+
+  useEffect(() => {
+    const getPhotos = async () => {
+      const photoListResponse = await getWithdrawalPhotosList(withdrawalId!!);
+      if (!photoListResponse.requestSucceeded || photoListResponse.error) {
+        setPhotoUrls([]);
+        snackbar.toastError(photoListResponse.error);
+      } else {
+        const photoUrlArray: string[] = [];
+        photoListResponse.photoIds?.forEach(({ id }) => {
+          photoUrlArray.push(
+            NURSERY_WITHDRAWAL_PHOTO_ENDPOINT.replace('{withdrawalId}', withdrawalId!!.toString()).replace(
+              '{photoId}',
+              id.toString()
+            )
+          );
+        });
+        setPhotoUrls(photoUrlArray);
+      }
+    };
+
+    if (withdrawalId) {
+      getPhotos();
+    }
+  }, [withdrawalId, snackbar]);
 
-export default function Photos(props: PhotosSectionProps): JSX.Element {
   return (
     <>
+      <ViewPhotosModal
+        photosUrls={photoUrls}
+        open={photosModalOpened}
+        onClose={() => setPhotosModalOpened(false)}
+        selectedSlide={selectedSlide}
+      />
       <Typography fontSize='20px' fontWeight={600}>
         {strings.PHOTOS}
       </Typography>
-      <Box>(Photos go here)</Box>
+      <Box display='flex' flexWrap='wrap'>
+        {photoUrls.map((photoUrl, index) => (
+          <Box
```

Comment:
@charlie-tf this looks great in staging. Can you add a 'cursor: pointer' style to this?

Reply:
can do!